tools/cli/domainUtils.go

```diff
@@ -299,7 +299,8 @@ func initializeDomainHandler(
 func initializeLogger(
 	serviceConfig *config.Config,
 ) log.Logger {
-	return loggerimpl.NewLogger(serviceConfig.Log.NewZapLogger())
+	zapLogger, _ := serviceConfig.Log.NewZapLogger()
```

Comment:
log fatal or panic here in case of error?

Reply:
I guess you don't have a logger, so the only thing you can do is to panic

Reply:
Yeah, I have used the `ErrorAndExited` to log the error and exit